packages/vsce/package.json

```
@@ -824,7 +826,9 @@
     "test:e2e": "npm run test:e2e:setup && npm run test:e2e:run && npm run test:e2e:teardown",
     "test:e2e:dev:interactive": "npm run test:e2e:setup && npx playwright test -c __tests__/__e2e__/playwright.config.ts --workers=1 --ui && npm run test:e2e:teardown",
     "test:e2e:dev": "npm run test:e2e:setup && npx playwright test -c __tests__/__e2e__/playwright.config.ts --workers=1 --headed && npm run test:e2e:teardown",
-    "pretty": "prettier --write \"src/**/*.ts\" \"__tests__/**/*.ts\""
+    "pretty": "prettier --write \"src/**/*.ts\" \"__tests__/**/*.ts\"",
+    "l10n:export": "mkdir -p dist && npx vscode-l10n-dev export \"src/**/*.ts\" --outDir l10n",
```

Should this be this?

```suggestion
    "l10n:export": "mkdir -p l10n && npx vscode-l10n-dev export \"src/**/*.ts\" --outDir l10n",
```